src/test/java/com/wikia/webdriver/testcases/logintests/LoginTests.java

```diff
@@ -36,14 +42,28 @@ public void Login_002_dropDownUser() {
     base.verifyUserLoggedIn(credentials.userName2);
   }
 
-  @Test(groups = {"Login_003", "Login"})
-  public void Login_003_specialPageStaff() {
+  @Test(groups = {"Login_003", "Smoke5"})
+  @Execute(onWikia = "agas")
+  public void Login_003_newAuthInGlobalNav() {
+    WikiBasePageObject base = new WikiBasePageObject(driver);
+    base.openWikiPage(wikiURL);
+    NavigationBar signInLink = new NavigationBar(driver);
+    signInLink.clickOnSignIn();
+    AuthModal newAuthModal = signInLink.getNewAuthModal();
+    Assert.assertTrue(newAuthModal.isOpened());
```

Comment:
Ok I see what happened. You renamed the AuthModal class according to my last comment, but didn't rename:
getNewAuthModal method
newAuthModal variable (inside AuthModal class)
and "newAuthModal" is also used in all the test names

Please adjust the names so that the "new" adjective is removed also from these places, in the context of auth modal. 

Please use refactoring renaming tool to save time (shift + f6)